plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -780,10 +796,52 @@
               powerlevel: 2
               channel_width: "40 MHz"
       register: output_list
+
+    - name: Updating Access Point Site / Configuration details
+      cisco.dnac.accesspoint_workflow_manager:
+        dnac_host: "{{ dnac_host }}"
+        dnac_username: "{{ dnac_username }}"
+        dnac_password: "{{ dnac_password }}"
+        dnac_verify: "{{ dnac_verify }}"
+        dnac_port: "{{ dnac_port }}"
+        dnac_version: "{{ dnac_version }}"
+        dnac_debug: "{{ dnac_debug }}"
+        dnac_log: True
+        dnac_log_level: DEBUG
+        config_verify: True
+        state: merged
+        config:
+          - mac_address: 6c:d6:e3:75:5a:e0
```

Comment:
Please note that when providing an example, it is good to use names that are easy to remember, such as "mac_address," rather than names that are difficult to remember. In the example, there are three options to update the access_point: mac_address, ap_name, and ip_address. Any of these options can be used, but it is easy for customers to give the ap_name. 

In the current document, we documented the MAC address first. And then, we are documenting the AP name but that is also not next to mac_address…management_ip_address is placed somewhere else.. 

I suggest to write ap_name first in your documentation.  Can you update?

Reply:
Identifier is mac_address or management_ip_address or hostname these param unchangeable,  but ap_name can be changed as a new ap_name so we cannot make ap_name to first.

Reply:
In the current document, we documented the MAC address first. And then, we are documenting the AP name but that is also not next to mac_address…management_ip_address is placed somewhere else.. Please make sure in document we do in order... MAC address, ip_address and AP name.